### PR TITLE
Docs: Update link to benchmark script

### PR DESCRIPTION
**Pull request by danieldiekmeier**
### What are you trying to accomplish?

The Performance section to the Docs links to a benchmark script. The script was recently renamed to `partial_benchmark.rb`: https://github.com/ViewComponent/view_component/commits/main/performance/partial_benchmark.rb

This PR updates the link!

### What approach did you choose and why?

Edited the markdown with the new link.

### Anything you want to highlight for special attention from reviewers?

Nope!

### CODE REVIEW

**Comment by Spone** [comment]
Thanks! Can you add a Changelog entry?

**Comment by danieldiekmeier** [comment]
Done!

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ nav_order: 5
 
     *Yoshiyuki Hirano*
 
+* Update link to benchmark script.
```
Comment (Spone) [inline docs/CHANGELOG.md:17]:
```suggestion
* Update link to benchmark script in docs.
```

**Review by Spone — APPROVED** [review]

File: docs/CHANGELOG.md (hunk 1 above)
Comment (danieldiekmeier) [reply docs/CHANGELOG.md]: Thanks for the change! I guess I was a little too brief.

**Pull request merged**